CONTRIBUTING.md

```diff
@@ -244,10 +225,8 @@ particular will be located at `docs/_build/html/release_notes.html`
 Please see the [Installing Qiskit Nature from
 Source](https://github.com/qiskit-community/qiskit-nature#installation)
 
-Note: Nature depends on Terra, and has optional dependence on Aer and IBM Q Provider, so
-these should be installed too. The main branch of Nature is kept working with those other element
-main branches so these should be installed from source too following the instructions at 
-the same location
+Note: Nature depends on Qiskit, and has optional dependence on Aer so
+should be installed too.
```

Comment:
```suggestion
Note: Nature depends on Qiskit, and has optional dependence on Aer so
these should be installed too.
```

We could be even more specific and say that Qiskit has to be installed and Aer may be, but I think that is not necessary.

Reply:
We had the same sort of text in all the repos and I left Aer there since its often used in tutorials and unit tests. I did not check explicitly as my main goal was just to switch to to Qiskit but with the repos moving to community I cleaned up the contributing where it seemed outdated but did not want to spend a bunch of time.

Since Nature explicitly depends on Qiskit installing Nature will install that by default. Hence the only optional install is Aer that you would need to do separately which is why it was worded like that. I had toyed with dropping the while text there but I figured using the Aer primitives people of do so I left it - and its in our requirements dev too.

Reply:
All of that is okay, but without some word (like `these` from my suggestion) this sentence does not work, does it?
```
and has optional dependence on Aer so should be installed too.
```
This is the current (sub-)sentence.